specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
+      "get": {
```

Comment:
The `/sip:checkNumberRouting` syntax suggests that this is a POST action, not a GET, see guideline https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action

Alternatively ,we could model this as a query param filter on a new GET route for TrunkRoutes. Something like GET /sip/routes?matches=+12345556789. But that would mean to introduce `/routes` and `/trunks` subcollection routes in addition to the `/sip` configuration object that contains both.

Reply:
How about `/sip/checkNumberRouting`? I don't want to introduce more subcollections for sip configuration - that would require overhead to support them in the future but here I just want to add the functionality for testing number pattern matching.

Reply:
How about `GET /sip/routesByPriority`?